ns1/rest/records.py

```diff
@@ -125,24 +133,21 @@ def create(
         )
 
     def create_raw(
-        self, zone, domain, type, body, callback=None, errback=None, **kwargs
+        self, z, domain, type, body, callback=None, errback=None, **kwargs
```

Comment:
i _had_ to rename this variable, because py doesn't like it when a kwarg and an arg match. For example passing `domain=` kwarg is not legal for this function

i swear it wasn't just because this improves the name by making it more ambiguous